oak_runtime/src/node/http/mod.rs

```diff
@@ -56,6 +60,16 @@ use tokio_rustls::TlsAcceptor;
 #[cfg(test)]
 pub mod tests;
 
+#[allow(clippy::enum_variant_names)]
+#[derive(Debug)]
+enum HttpError {
+    ChannelError(String),
```

Comment:
Would it make sense to have a variant that just wraps https://docs.rs/http/0.2.1/http/struct.Error.html , and also implement `From<Error>` for this struct? Then you could just use `?` to propagate the underlying errors.

Reply:
If these variants are only ever used for logging, then arguably there is not much point in preserving the variants at all, right? May as well use `anyhow::Result` or something like that that just allows logging. Structured errors are usually only relevant if you expect the caller to do something different based on the actual error, e.g. retry vs abort. Something to revisit at some point in the future anyways.

Reply:
Yes. The idea of using `anyhow` came up before too. But `anyhow::Context` is only implemented for the Result types that use StdError as the error type. To be able to use `anyhow::Context` here, we have to wrap the rest of the errors (e.g., OakStatus, ConfigurationError) in an `anyhow::Error`. So it won't be as succinct as you'd expect. Or is there a better way to do this?  

I have created #1693 to track this.